### DISCUSSION TITLE
Apprise not working?

### DISCUSSION THREAD

**Discussion by jackblk** [nikhilbadyal/docker-py-revanced#433] [Q&A]
Hi, I tried to use Apprise but it doesn't post anything to me yet. I checked the logs and there's no log about using apprise as well.

Here's my command:

```shell
docker run --rm -it \
	-v "$(pwd)"/tmp/apks:/app/apks/ \
	-e PATCH_APPS=youtube \
	-e YOUTUBE_CLI_DL=https://github.com/inotia00/revanced-cli \
	-e YOUTUBE_PATCHES_DL=https://github.com/inotia00/revanced-patches \
	-e YOUTUBE_PATCHES_JSON_DL=https://github.com/inotia00/revanced-patches \
	-e YOUTUBE_INTEGRATIONS_DL=https://github.com/inotia00/revanced-integrations \
	-e APPRISE_URL=tgram://xxxx:aaaa/yyyy \
	nikhilbadyal/docker-py-revanced
```

I tried adding `APPRISE_NOTIFICATION_TITLE` but it's not working as well. Any solution?

**Comment by nikhilbadyal** [comment]
Works only when Building with [Github](https://github.com/nikhilbadyal/docker-py-revanced/blob/f5053cdd60eb35c944594efa8e24df15433723ac/.github/workflows/build-apk.yml#L135).

**Comment by nikhilbadyal** [comment]
Also apprise used Bot API for telegram which doesn't allow sending file >50MB.

**Reply by jackblk** [reply]
Quite disappointing but I understand.
